src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -77,6 +77,8 @@ PrintHelp(
         "  -platency<0/1>           Print latency statistics. (def:0)\n"
         "\n"
         "  Scenario options:\n"
+        "  -scenario:<profile>      Scenario profile to use.\n"
+        "                            - {upload, download, hps, rps-single, rps-multi, latency}.\n"
```

Comment:
Bikeshed: Should we standardize on naming to be {scenario}, {scenario}-multi to indicate single connection vs multi connection versions, or {scenario}-single, {scenario}-multi? Because I was thinking we should have multi connection throughput and possible latency tests too....

I think I like {scenario}, {scenario}-multi

Reply:
In other words, thoughts on:
```
{upload, upload-multi, download, download-multi, hps, rps, rps-multi, latency, latency-multi}
```